dist/jsonapi-client-test.js

```diff
@@ -454,6 +468,10 @@ Transport.prototype._action = function(method, url, data, callback) {
       payload.body = JSON.parse(payload.text);
```

Comment:
Out of scope of this change, but is there any danger `JSON.parse(payload.text)` will fail to parse and throw and exception? Should we put in a `try..catch` block?